### ISSUE TITLE
Parse error: syntax error, unexpected '.', expecting '&' or variable

### ISSUE THREAD

**Issue by ghost** [dshafik/php7-mysql-shim#112]
Parse error: syntax error, unexpected '.', expecting '&' or variable (T_VARIABLE) in /opt/cpanel/ea-php73/vendor/dshafik/php7-mysql-shim/lib/mysql.php on line 551

**Comment by dshafik** [comment]
Have you updated to the latest version?

Can you paste line 551 from your file here? It doesn't look incorrect to me.

**Comment by ghost** [comment]
We just installed a few days ago and can't seem to get rid of this error.  Here is line 551.

function mysql_fieldname(... $args)

**Comment by dshafik** [comment]
So the issue is here is that you've installed what looks to be v0.2.0, when the latest is 1.0.0beta4.

I suspect you either manually chose to not install a beta, assuming 0.2.0 was more stable, or composer did. This is incorrect, in semantic versioning, all < 1.0 are unstable. Install 1.0.0beta4 and try again.

Also it's worth noting that you're using PHP 7.3, which has been unsupported and EOL for over two and a half years. And trying to use a library that is no longer actively maintained, that was always meant to be a temporary solution in the first place.

**Comment by ghost** [comment]
You are correct. Composer defaulted to 0.2.0. When I specified the version, the issue was resolved.

Running php5.4 in this environment and have thousands of scripts. Rector doesn’t seem to cover MySQL any longer so we have some work to do…. Any suggestions?

Thanks for your help!

Andrew Harrod - 
K6ARH
RFinder LLC
e: ***@***.*** | w: www.rfinder.com
________________________________
